src/jace/util/jax_helper.py

```diff
@@ -258,3 +264,42 @@ def get_jax_literal_value(lit: jax_core.Atom) -> bool | float | int | np.generic
     if util.is_scalar(val):
         return val
     raise TypeError(f"Failed to extract value from '{lit}' ('{val}' type: {type(val).__name__}).")
+
+
+def parse_backend_jit_option(
+    backend: str | dace.DeviceType,
```

Comment:
As mentioned above, I'd rather:
```suggestion
    backend: Literal['cpu', 'gpu'],
```

Reply:
See above.